docs/excel/custom-functions-troubleshooting.md

```diff
@@ -0,0 +1,65 @@
+
+Excel has a number of built-in error messages which are returned to a cell if there is calculation error. These are `#NULL!`, `#DIV/0!`, `#VALUE!`, `#REF!`, `#NAME?`, `#NUM!`, `#N/A`, and `#GETTING_DATA`. At this time, custom functions can only use the error codes listed above when custom functions hit an error.
+
+## Common issues
+
+### Verify SSL certificates
+
+If your add-in fails to register, [verify that the SSL certificates are configured correctly](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md) for the web server that's hosting your add-in application. Typically if you forget to do this step, you will see an error message in Excel warning you that custom functions could not be installed properly. For more information on this verification, see [Adding self-signed certificates as trusted root certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md).
+
+### Associate your functions
+
+In your custom functions' script file, you need to both write the function code and code to associate the function's name to the corresponding JSON id. Typically this declaration is made after each function or at the end of the script file. It's common to forget to include this and can be a reason why your functions don't work.
+
+The following example shows an add function, followed by the function's name `add` being associated with the corresponding JSON id `ADD`.
+
+```js
+function add(first, second){
+  return first + second;
+}
+
+CustomFunctions.associate("ADD", add);
+```
+
+For more information on this process, see [Associating function names with json metadata](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-best-practices#associating-function-names-with-json-metadata).
+
+### Ensure promises return
+
+In addition to the ordinary reasons that a cell might report a #GETTING_DATA error, custom functions also will report `#GETTING_DATA` if a promise does not return a result. Check your functions to make sure that any promises are properly returning a result to a cell.
+
+## Reporting Feedback
+
+If you are encountering errors that aren't documented here, let us know. The custom functions and documentation teams have several procedures in place for reporting issues.
+
+### In Excel on Windows or Mac
+
+If using Excel for Windows or Mac, you can report feedback to the Excel Custom Functions team directly in the UI of Excel itself. To do this, select **File | Feedback | Send a Frown**. Sending a frown will provide the necessary logs to understand the issue you are hitting.
+
+### In Github
+
+Feel free to submit an issue you encounter either through the "edit" feature on any documentation page, by submitting a [pull request to the office js docs repository](https://github.com/OfficeDev/office-js-docs-pr/blob/master/Contributing.md), or by [filing a new issue directly to the custom functions repository](https://github.com/OfficeDev/Excel-Custom-Functions/issues).
```

Comment:
```suggestion
Feel free to submit an issue you encounter either through the "Content feedback" feature at the bottom of any documentation page, or by [filing a new issue directly to the custom functions repository](https://github.com/OfficeDev/Excel-Custom-Functions/issues).
```
Took out PR instructions because likely they don't know the fix if they are reporting an issue.

Reply:
Makes sense!